spec/person_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rspec'
+require './book'
+require './rental'
+require './person'
+describe Person do
+  before(:each) do
+    @person = Person.new(20, 'John')
+  end
+  context 'when creating a new book' do
+    it 'should create a new person' do
+      person = Person.new(20, 'John')
```

Comment:
- [ ] Since you're creating a person inside the `before :each` hook, you can safely remove this line of each spec to keep your code clean and follow the DRY principle, (Don't forget to add the `@` to person)! 🤓 🔧